test/js/data/symbol_bucket.test.js

```diff
@@ -28,7 +28,7 @@ test('SymbolBucket', function(t) {
     }
 
     function bucketSetup() {
-        var bucket = new SymbolBucket(buffers, info, 1);
+        var bucket = new SymbolBucket(buffers, info, 1, 0);
```

Comment:
Huh. Weird it was passing with this as `undefined` before. I agree that it looks necessary though.